source/install/desktop-custom-dictionaries.rst

```diff
@@ -0,0 +1,33 @@
+Desktop Custom dictionaries
+===========================
+
+Mattermost Desktop allows users to download their dictionary definitions from a custom URL. If the user doesn't want to provide
+such a service, dictionaries definitions will be automatically obtained from Chromium's CDNs.
+
+This feature enables users to skip using Chromium's CDNs as well as to provide customized dictionaries definitions.
+
+This feature is available since v4.7.1 on Windows and Linux.
+
+Setting up the dictionaries
+---------------------------
+
+The desktop app uses [Hunspell Dinctionary definitions](https://hunspell.github.io/) for Windows and Linux (for macOS definitions 
+are provided by the OS itself). A quick way to get them is to obtain a copy of `hunspell_dictionaries.zip` from the [latest 
+electron release](https://github.com/electron/electron/releases/latest).
+
+Once obtained it needs to be extracted and converted to lowercase, as chromium won't try to access `en-US...` but `en-us...`.
```

Comment:
```suggestion
Once downloaded, the dictionary definitions need to be extracted and converted to lowercase because chromium expects `en-us` rather than `en-US...`.
```

Reply:
Does it make sense to mention that once the default dictionary definitions are downloaded, users can then modify them as needed with their own definitions?

Reply:
i was just trying to provide a minimum working solution, but I guess that's doable as well :)